packages/s/shepherd.js.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "shepherd.js",
+  "description": "Guide your users through a tour of your app.",
+  "keywords": [
+    "site tour",
+    "tour",
+    "tutorial",
+    "shepherd"
+  ],
+  "authors": [
+    "Adam Schwartz <adam.flynn.schwartz@gmail.com>",
+    "Zack Bloom <zackbloom@gmail.com>"
+  ],
```

Comment:
I don't think we really support displaying `authors` anywhere. Might be better to make this a single string for the `author` attr?

Reply:
@MattIPv4 what is the author info used for? Should it be someone who is actively working on the project? The original authors have been long gone.

Reply:
They're used in package metadata on the website and will be displayed on the library page for the new website as the authors.

(Zack is still around, he now helps run cdnjs at Cloudflare).

Reply:
@MattIPv4 I wasn't saying they weren't working, just that they are not involved in Shepherd.

Reply:
Please advise if it would be best to set it to one of the original authors to show they were the authors or to set it to myself as a current maintainer. Happy to do either.

Reply:
I think it'd be est to set yourself as the author if you're the one maintaining it now 👍